### PR TITLE
Update dev env chapter

### LINKED ISSUE #5: Complete ARCHER2 development environment
Complete the following tasks:
- [x] Add in the set of default modules once known.
- [x] Check this name (this = PrgEnv-aocc).
- [x] Add output in _Switching between compiler environments_ section.
- [x] Add output in _Switching between different compiler versions_ section.

### PR DESCRIPTION
**Pull request by juanfrh**
Command output added.
AMD compiler removed (as it's not available on the TDS).
Information about compiler wrappers added.

### CODE REVIEW

File: _episodes/03-develop-env.md [hunk 1]
```diff
@@ -157,77 +208,97 @@ The Lmod tool will load these dependencies for you when you load the module (see
...
 If you want to swap two versions of the same module then you simply load the version that you 
 want to swap for the currently loaded version, Lmod recognises that they are the same module 
-with different versions and swaps them for you. 
+with different versions and swaps them for you.
 
 ## Switching between compiler environments
 
-You use the `module swap` command to switch between the three different compiler envionments available
+You use the `module swap` command to switch between the three different compiler environments available
 on ARCHER2. The available environments are:
```
Comment (lcebaman) [inline _episodes/03-develop-env.md:275]: two different compilers now until the AMD ones are available

**Review by lcebaman — CHANGES_REQUESTED** [review]
We should probably change the text where it says three different compilers to two different compilers until the AMD ones are installed.

File: _episodes/03-develop-env.md (hunk 1 above)
Comment (juanfrh) [reply _episodes/03-develop-env.md:275]: Thanks for spotting that. I've updated the number to "two".

**Review by lcebaman — APPROVED** [review]
Looks good

**Pull request merged**